src/veme.c

```diff
@@ -1,5 +1,6 @@
 #include <stdio.h>
 #include <stdlib.h>
+#include "veme.h"
```

Comment:
veme.h is deprecated, we should probably be using `meme.h` here

Reply:
i'll give you 4 scrum points if you ignore this

Reply:
no one needs to know about this